x/concentrated-liquidity/internal/math/math.go

```diff
@@ -113,7 +113,7 @@ func GetNextSqrtPriceFromAmount0RoundingUp(sqrtPriceCurrent, liquidity, amountRe
 // getNextSqrtPriceFromAmount1RoundingDown utilizes the current squareRootPrice, liquidity of denom1, and amount of denom1 that still needs
 // to be swapped in order to determine the next squareRootPrice
 func GetNextSqrtPriceFromAmount1RoundingDown(sqrtPriceCurrent, liquidity, amountRemaining sdk.Dec) (sqrtPriceNext sdk.Dec) {
-	return sqrtPriceCurrent.Add(amountRemaining.Quo(liquidity))
+	return sqrtPriceCurrent.Add(amountRemaining.QuoTruncate(liquidity))
```

Comment:
Does this mean that we want to truncate instead of doing baker's rounding?

Reply:
Yes, that's correct.

Rounding down was the intended behavior given the method name.

Additionally, I've run into panics while testing fees stemming from the fact that we do not round down / truncate here.

Reply:
ACK On this change